jsonrpc/dispatcher.go

```diff
@@ -295,7 +296,7 @@ func (d *Dispatcher) handleReq(req Request) ([]byte, Error) {
 	output := fd.fv.Call(inArgs)
 	if err := getError(output[1]); err != nil {
 		d.logInternalError(req.Method, err)
-		return nil, NewInternalError("Internal error")
+		return nil, NewInvalidRequestError(err.Error())
 	}
```

Comment:
Ganache 

> Wasn't this task supposed to:
> 
> ```
> Review all JSON RPC API methods and update the error response messages to meet infura standard's.
> ```
> 
> I don't see many response messages updated, but I may be wrong.
> 
> _Note:_ Please, in the future, let's not reuse existing branches for new PRs (this one was used for [PR#138](https://github.com/0xPolygon/polygon-sdk/pull/138)). Let's always open a new branch for a new PR, otherwise, GitHub tends to get funny with displayed changes.

@Vuksan most of the error messages are aligned, only few need to be update, I have done that.

Reply:
It should be internal error optimally, but ganache is returning an invalid request error.